engine/src/cmd/collide2/Ice/IcePoint.h

```diff
@@ -207,14 +207,17 @@
 		//! Clamps each element
 		inline_	Point&			Clamp(float min, float max)
 								{
-									if(x<min)	x=min;
-			                                                   else if(x>max)	x=max;
-									if(y<min)	y=min;
-			                                                    else if(y>max)	y=max;
-									if(z<min)	z=min;
-			                                                     elseif(z>max)	z=max;
-									return *this;
-								}
+                                    if(x<min) x=min;
+                                    else if(x>max) x=max;
+
+                                    if(y<min) y=min;
+                                    else if(y>max) y=max;
+
+                                    if(z<min) z=min;
+                                    else if(z>max) z=max;
```

Comment:
These probably need to be in a min-max function:
x = min <= x <= max

So:
if x < min: x = min
if x > max: x = max
or something like:
```
x = std::min(x, std::max(x, max))
y = std::min(y, std::max(y, max))
z = std::min(z, std::max(z, max))
```

Can probably define a `minmax()` function to make it more obvious and less typing.

Reply:
Or see https://en.cppreference.com/w/cpp/algorithm/minmax

Reply:
Hi @BenjamenMeyer, I see such things in a lot of places in the code. I want to replace all of those places with a generic function that will be defined once for the entire codebase. I'll do that in a separate PR.